app/uk/gov/hmrc/exports/models/DeclarationSort.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.models
+
+case class DeclarationSort(by: String = "createdDateTime", direction: Int = DeclarationSort.ASC)
```

Comment:
It'd be nice if we could keep the `by` and `direction` fields in an enum, that way we can protect against sorting on unexpected fields.

e.g. if I try to sort on "id" or something, the binder might throw a 400 or just use the default sort order

Reply:
Is that because sorting on any random field is in-efficient without an index on that field?

Reply:
Implementation of `QueryStringBindable` type class would be nice

Reply:
> Is that because sorting on any random field is in-efficient without an index on that field?

Yehh there is that, but also for a validation point of view